packages/zed-js/src/client/base-client.ts

```diff
@@ -145,18 +147,7 @@ export abstract class BaseClient {
   ${this.baseURL}/query`;
   }
 
-  protected async send(opts: {
-    method: 'GET' | 'POST' | 'DELETE' | 'PUT';
-    path: string;
-    body?: Types.IsoBody;
-    format?: Types.ResponseFormat;
-    signal?: Types.IsoAbortSignal;
-    headers?: Record<string, string>;
-    timeout?: number;
-    contentType?: string;
-    duplex?: 'half';
-    dontRejectError?: boolean;
-  }) {
+  protected request(opts: Types.RequestOpts) {
```

Comment:
I separated the creation of the request into this request method. The existing `send` method then calls this.

The `describeQuery` method only uses `request` and handles errors in a different way than the other methods.